frontend/src/routes/(authenticated)/admin/EditUserAccount.svelte

```diff
@@ -16,15 +16,23 @@
   export let currUser: LexAuthUser;
   export let deleteUser: (user: User) => void;
 
-  const schema = z.object({
+  const schema = z
+    .object({
     email: z.string().email($t('form.invalid_email')).nullish(),
     name: z.string(),
     password: passwordFormRules($t).or(emptyString()).default(''),
     score: z.number(),
     role: z.enum([UserRole.User, UserRole.Admin]),
-  });
+    })
+  const refinedSchema = schema
+    .refine((data) => data.email || data.role !== UserRole.Admin, {
+    message: 'You must have a valid email.',
```

Comment:
This needs to go in en.json and be translatable. Also it should not refer to "you": the person who needs the valid email is the user being edited, NOT the admin doing the editing (who already has a valid email)..